### DISCUSSION TITLE
InputNumber组件，可以拷贝汉字提交到Form表单

### DISCUSSION THREAD

**Discussion by cx5168** [ant-design/ant-design#33587] [Q&A]
antd: v4.18.2
reactjs: v17.0.2
windows: v10
browser: edge v96

form表单里描画inputnumber组件，通过拷贝黏贴输入汉字，虽然画面上失去焦点时不会显示汉字，但是提交表单时，会将输入的汉字提交上去。

这算组件的bug吗？

**Comment by afc163** [comment]
给个重现。
